arangod/Aql/TraversalExecutor.cpp

```diff
@@ -840,6 +840,7 @@ bool TraversalExecutor::initTraverser(AqlItemBlockInputRange& input) {
       std::string sourceString;
       TRI_ASSERT(_inputRow.isInitialized());
 
+      _traversalEnumerator.unprepareValidatorContext();
```

Comment:
Oh, good catch!
I think it did not matter thus far, as we have always override the variables.